src/extension.ts

```diff
@@ -380,7 +388,6 @@ async function setupContextValues() {
   // allow for easier matching using "in" clauses for our Resources/Topics/Schemas views
   const viewsWithResources = setContextValue(ContextValues.VIEWS_WITH_RESOURCES, [
     "confluent-resources",
-    "new-confluent-resources",
```

Comment:
This hasn't existed for a while. Current views:
https://github.com/confluentinc/vscode/blob/b3b9952ce773ae9c1da152e2bf8cd9ae6b3d618a/package.json#L1060-L1101